src/lib/pbo/PBO.jl

```diff
@@ -2,10 +2,10 @@ module PBO
 
 using Random
 
-# -*- Imports -*-
-import QUBOTools: varcmp, qubo, variables, variable_map, variable_set, variable_inv
+#  Imports 
+import QUBOTools: varlt, qubo, variables, variable_map, variable_set, variable_inv
 
-# -*- Exports -*-
+#  Exports 
 # export PBF
```

Comment:
Do we need these comments? If so, maybe add the equation in this header to know what the symbols mean

Reply:
Removed